### DISCUSSION TITLE
How can I make dropdowns not clip like this

### DISCUSSION THREAD

**Discussion by TreyWW** [saadeghi/daisyui#2575] [Q&A] [answered]
Hi there,
So I have this dropdown, at the moment I have logic so if its the last row in the table, it'll use "dropdown-top" which works fine for the bottom element. But now the second to last element still gets clipped too. How can I make sure it stays on top? I'm guessing its possibly the zindex, but ive been messing around with it and nothing has changed. Just wondering how to make sure this doesn't keep getting clipped so either `auto position at top if it cant fit`, or `sit  on top of all elements`

Looking again it looks like it's the overflow, the table is set to `overflow-y-hidden`. I don't want a scroll bar, i Just want the dropdown to be on top (not hidden).
 Tried adding overflow-visible to the actual dropdown but still no luck

<img width="138" alt="image" src="https://github.com/saadeghi/daisyui/assets/73353716/0b3c7dbb-f5a0-4806-89c1-9ba60a4ad6cd">

https://play.tailwindcss.com/mp7IhUg9c0

Thanks :)

**Comment by saadeghi** [comment, marked as answer]
It's not about z-index. It's only because those overflow hidden classes you have.  
Overflow hidden clips everything inside so even a dropdown can't overflow.  
It's better if you use the overflow scroll on a parent element that already can fit the dropdown menu.

Another option would be using a JS library like [floating UI](https://floating-ui.com/) which is designed to fix this exact problem. It makes sure your dropdown or tooltip, etc is always inside the screen. It automatically changes the position and makes sure it doesn't overflow.

**Reply by TreyWW** [reply]
Cool thank you, yeah I'm going to make it scroll for now and then try and find a way to make it just scale down rather th an scroll. Almost need to set the max-h or something maybe. At least i know where to look now, thanks
